doc/cloud_init.rst

```diff
@@ -3,106 +3,39 @@
 Cloud-Init Integration
 ======================
 
-commissaire provides a script in ``tools/cloud-init/part-handler.py``
-to help automatically register hosts to the commissaire server during
-bootup by way of cloud-init.
+Commissaire provides a ``commctl`` command to generate ``user-data`` file
```

Comment:
clarify: ... to generate __a__ ``user-data`` file __for cloud-init__ that automatically ...